composer.json

```diff
@@ -63,8 +63,12 @@
     },
     "autoload" : {
         "psr-4" : {
-            "oat\\taoLti\\" : "",
-            "OAT\\Library\\Lti1p3Core\\Tests\\": "../vendor/oat-sa/lib-lti1p3-core/tests/"
+            "oat\\taoLti\\" : ""
         }
+    },
+    "autoload-dev" : {
```

Comment:
this was added as part of the previous PR by Mathieu. Not sure why this diff is in this PR as well, something wrong with branches?

Reply:
Do not see https://github.com/oat-sa/extension-tao-lti/pull/300/files